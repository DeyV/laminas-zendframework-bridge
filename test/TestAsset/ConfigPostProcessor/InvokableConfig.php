<?php
return array(
    'dependencies' => array(
        'invokables' => array(
            'console-helper'                 => 'Zend\Stdlib\ConsoleHelper',
            'Some\Custom\ClassName'          => 'Zend\Stdlib\ConsoleHelper',
            'Zend\Stdlib\StringWrapper\Intl' => 'Zend\Stdlib\StringWrapper\Intl',
        ),
    ),
);
