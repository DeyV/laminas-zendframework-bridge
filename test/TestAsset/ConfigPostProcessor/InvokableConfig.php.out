<?php
return array(
    'dependencies' => array(
        'invokables' => array(
            'console-helper'                 => 'Laminas\Stdlib\ConsoleHelper',
            'Some\Custom\ClassName'          => 'Laminas\Stdlib\ConsoleHelper',
            'Zend\Stdlib\StringWrapper\Intl' => 'Laminas\Stdlib\StringWrapper\Intl',
        ),
    ),
);
